R/input-comparison.R

```diff
@@ -39,15 +39,26 @@ prepare_art_spectrum_comparison <- function(art, shape, pjnz) {
   if(identical(unique(art_single_cq$sex), c("both"))) {
     # If no sex aggregated data present in ART data, aggregate Spectrum by age
     spec_aggreagted <- pjnz$art_dec31 |>
-      dplyr::count(spectrum_region_code, year, age_group,
-                   wt = art_dec31, name = "value_spectrum") |>
-      dplyr::mutate(sex = "both")
-
+      dplyr::mutate(calendar_quarter = paste0("CY", year, "Q4")) |>
+      dplyr::group_by(spectrum_region_code, calendar_quarter, year, age_group) |>
+      dplyr::summarise(
+        value_spectrum = sum(art_dec31),
+        artadj_factor = sum(artadj_factor),
+        artadj_absolute = sum(artadj_absolute),
+        .groups = "drop") |>
+      dplyr::mutate(
+        value_spectrum_adjusted = (value_spectrum + artadj_absolute) * artadj_factor,
+        artadj_factor = dplyr::if_else(age_group == "Y000_014",artadj_factor, artadj_factor/2),
```

Comment:
What is the `/2` in this in this line for?

Reply:
It was to recalculate the % adjustment factor after aggregating both sexes on ART (edge case for when a user provide adult ART data for all sexes in Naomi) but no longer relevant with change to % adjustment calculated from `art_dec31_reported`, `art_dec31_attend` and
`art_dec31_reside`